### PR TITLE
Update Image to 2.4.0 and did correction in Link

### PR DESCRIPTION
**Pull request by Kartik494**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```
NONE
```

### CODE REVIEW

**Comment by Kartik494** [comment]
Hi i have updated the existing image to `quay.io/external_storage/local-volume-provisioner:v2.4.0` and update the Link for `update PV to beta` 

Thanks

File: deployment/kubernetes/example/default_example_provisioner_generated.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ spec:
     spec:
       serviceAccountName: local-storage-admin
       containers:
-        - image: "quay.io/external_storage/local-volume-provisioner:v2.1.0"
+        - image: "quay.io/external_storage/local-volume-provisioner:v2.4.0"
```
Comment (msau42) [inline deployment/kubernetes/example/default_example_provisioner_generated.yaml:39]: we've moved our image building to k8s.gcr.io now: k8s.gcr.io/sig-storage/local-volume-provisioner:v2.4.0

Comment (Kartik494) [reply deployment/kubernetes/example/default_example_provisioner_generated.yaml]: Hi @msau42  I have addressed the changes .PTAL

**Pull request merged**